src/gcp_scanner/crawler/crawler_factory.py

```diff
@@ -55,6 +56,7 @@
   "static_ips": ComputeStaticIPsCrawler,
   "subnets": ComputeSubnetsCrawler,
   "sourcerepos": CloudSourceRepoCrawler,
+  "bq": BigQueryCrawler,
```

Comment:
please make sure they are in alphabetical order.

Reply:
done :+1: